UnitySDK/Assets/ML-Agents/Editor/Tests/MLAgentsEditModeTest.cs

```diff
@@ -111,12 +110,12 @@ public void TestAgent()
 
     public class EditModeTestInitialization
     {
-/*
-        private Brain GenerateTestBrain()
-        {
-            return ScriptableObject.CreateInstance<TestBrain>();
-        }
-*/
+        /*
```

Comment:
Why is this commented out, but not removed?

Reply:
Actually, only the tabs were changed. I do not know why this code is commented so I will remove it